config.js

```diff
@@ -2,13 +2,4 @@
 angular.module('main')
   .constant('st2Config', {
 
-    hosts: [{
```

Comment:
hm.. may be comment them out? Users need hints. Discoverability is a problem when we have "nothing".